extensions/utils/VaultClientWrapper.js

```diff
@@ -128,7 +141,7 @@ class VaultClientWrapper {
     }
 
     tempCredentialsReady() {
-        if (this._authConfig.type !== authTypeAssumeRole) {
+        if (this._authConfig.type !== authTypeAssumeRole || !this._useTempCredentials) {
```

Comment:
can't we rely only on `type !== authTypeAssumeRole`, instead of introducing this extra `useTempCredentials` ?

in S3C it should not be assume role anyway (since it is not supported), so we should be fine...

Reply:
S3C does actually support assumeRole and we do have that in the config of S3C as well. I introduced a new type of authentication (no auth) for the getAccounts API.